app/controllers/pages_controller.rb

```diff
@@ -0,0 +1,16 @@
+class PagesController < ApplicationController
+  def about
+  end
+
+  def contact
+    if request.post? # Check if it's a POST request
+      name = params[:name]
+      email = params[:email]
+      message = params[:message]
```

Comment:
It might be worthwhile to add strong params to this controller as well to conform to how your other controllers are written, but since the info from the params is only being displayed in a string and not being used any other way, it's not a needed change by any means.